test/experiment/ab_test.rb

```diff
@@ -37,15 +37,18 @@ def test_requires_at_least_two_alternatives_per_experiment
     assert_raises RuntimeError do
       new_ab_test :none do
         alternatives []
+        default nil
```

Comment:
They're not necessary, strictly speaking, but providing them prevents a warning from being issued. I can remove the default nil lines, it will just increase the chattiness of the tests. 

> On Jan 10, 2016, at 6:07 PM, Phillip Baker <notifications@github.com> wrote:
> 
> In test/experiment/ab_test.rb:
> 
> > @@ -37,15 +37,18 @@ def test_requires_at_least_two_alternatives_per_experiment
> >      assert_raises RuntimeError do
> >        new_ab_test :none do
> >          alternatives []
> > +        default nil
> Are these new defaults necessary? Won't it use the first alternative if none is specified?
> 
> —
> Reply to this email directly or view it on GitHub.
>